core/arch/arm/mm/core_mmu.c

```diff
@@ -79,6 +81,9 @@ static struct memaccess_area nsec_shared[] __nex_data = {
 };
 
 #if defined(CFG_SECURE_DATA_PATH)
+static const char *tz_sdp_match = "sdp";
+static struct memaccess_area sec_sdp = MEMACCESS_AREA(0, 0);
```

Comment:
right it is part of bss segment 
0e18cf30  O .bss	00000008 sec_sdp

but it could be easier to read the code without uninitialized variables

Reply:
> but it could be easier to read the code without uninitialized variables

That's the point, they are already initialized if they are static. Here we're normally depending on the implicit zero-initialization. We do have some exceptions, like initialization of spinlocks.

Reply:
ok